src/utils.test.js

```diff
@@ -100,18 +100,18 @@ describe('utils', () => {
           }),
         { initialProps: { delay: 100 } },
       );
-      // Act
+      // Assert
       jest.advanceTimersByTime(600);
       rerender({ delay: null });
       jest.advanceTimersByTime(800);
-      // Assert
```

Comment:
these changes are not related to this feature, right?

Reply:
No but they are related to this file, it's a pity not to fix something so trivial.